accessibility_monitoring_platform/apps/common/models.py

```diff
@@ -158,3 +158,12 @@ class Meta:
     def save(self, *args, **kwargs):
         self.version += 1
         super().save(*args, **kwargs)
+
+
+class UserCacheUniqueHash(models.Model):
+    """
+    Model subclassed to add versioning
```

Comment:
This docstring seems wrong. The model doesn't have versioning. Also, shouldn't explain what the model is for?